scripts/unsupported/revert_sam/SamToUnmappedBams.wdl

```diff
@@ -0,0 +1,150 @@
+task CramToBam {
+  File? input_cram
```

Comment:
hmm i think i know why all of these inputs are optional but it's so counterintuitive...lets talk about this (to make sure i understand)

Reply:
Yeah this isn't super user friendly, but I'm not sure how to make it less ugly.